src/components/Dropdown/styles.js

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from 'react-native';
+import common from 'constants/commonStyles';
+import { BLACK } from 'constants/colors';
+import typography from 'constants/typography';
+
+export const styles = StyleSheet.create({
+  inputIOS: {
```

Comment:
I think you could use `Platform` here
https://reactnative.dev/docs/platform-specific-code